pkg/backup/s3/s3.go

```diff
@@ -104,16 +109,16 @@ func (s *S3) List() ([]string, error) {
 func (s *S3) list(prefix string) ([]string, error) {
 	resp, err := s.client.ListObjects(&s3.ListObjectsInput{
 		Bucket: aws.String(s.bucket),
-		Prefix: aws.String(prefix),
+		Prefix: aws.String(path.Join(v1, prefix)),
```

Comment:
This is wrong..
prefix ends with "/".
With path.Join(), the ending "/" is gone.